elyra/pipeline/processor_local.py

```diff
@@ -233,9 +238,19 @@ def process(self, operation: Operation):
         envs.update(operation.env_vars_as_dict())
         t0 = time.time()
         try:
-            run(argv, cwd=file_dir, env=envs, check=True)
+            run(argv, cwd=file_dir, env=envs, check=True, stderr=PIPE)
+        except CalledProcessError as cpe:
+            error_msg = str(cpe.stderr.decode())
+            self.log.error(f'Internal error executing {file_name}: {error_msg}')
+
+            error_trim_index = error_msg.rfind('\n', 0, error_msg.rfind('Error'))
+            if error_trim_index != -1:
+                raise RuntimeError(error_msg[error_trim_index:]) from cpe
+            else:
+                raise RuntimeError('Internal error executing Python script') from cpe
```

Comment:
We should include the actual script name.
```suggestion
                raise RuntimeError(f'Internal error executing {file_name}') from cpe
```